### PR TITLE
Updated in subsections of Data-Only Hotspots

### PR DESCRIPTION
**Pull request by Dumpling-girl**
Updated onboarding tutorials under Data-Only Hotspots section. Checked weblinks and updated them where needed.
Changed links worded with 'here'.
Corrected spelling where needed.

#Solana Scribes 2024

### CODE REVIEW

**Review by waveform06 — COMMENTED** [review]
do data only hotspots still need to open port 1680?
balena doc refers to at the moment as Summer 2021 - we need today's moment

**Comment by Dumpling-girl** [comment]
Will do edits tomorrow.

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 id: balena-data-only-hotspot
 title: Data-Only Hotspot with balena + Raspberry Pi + RAK Concentrator RAK2287
-pagination_label: Balena Data-Only Tutorial
-sidebar_label: Balena Data-Only Tutorial
+pagination_label: balena Data-Only Tutorial
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: This should remain capitalized

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx [hunk 2]
```diff
@@ -15,11 +15,11 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
 
 <LegacyContentBanner />
 
-A data-only Hotspot is a type of Hotspot that participates on the Helium Network by transferring
-data from the end devices (LoRaWAN nodes). The data-only Hotspots are eligible for Network Data
-Transfer Rewards but not Proof of Coverage Rewards.
+A Data-Only Hotspot is a type of Hotspot that participates on the Helium Network by transferring
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: doesn't not need to be capitalized

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx [hunk 3]
```diff
@@ -15,11 +15,11 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
...
+data from the end devices (LoRaWAN nodes). The Data-Only Hotspots are eligible for Network
+Data Transfer rewards but not Proof-of-Coverage rewards.
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: data transfer does not need to be capitalized

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx [hunk 4]
```diff
@@ -30,8 +30,8 @@ installing the **_gatewayrs_** and **_packet forwarder_** services open sourced
 
 - Raspberry Pi 0/2/3/4 or balenaFin
 - SD card in case of the Raspberry Pi
-- [RAK 2287 LoRa concentrator SPI based](https://store.rakwireless.com/products/rak2287-lpwan-gateway-concentrator-module)
-  and [RAK2287 Pi Hat](https://store.rakwireless.com/products/rak2287-pi-hat)
+- [RAK 2287 LoRa concentrator SPI based](https://store.rakwireless.com/products/wislink-concentrator-module-sx1302-rak2287-lorawan?variant=41826859385030)
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: Old link works. why change it?

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx [hunk 5]
```diff
@@ -47,16 +47,17 @@ with the SPI interface.
...
 ## Deploy the code
 
-Running this data-only Hotspot with balena.io is as simple as deploying it to a balenaCloud
+Running this Data-Only Hotspot with balena.io is as simple as deploying it to a balenaCloud
 application. You can do it in just one click by using the button below:
 
 [![](https://www.balena.io/deploy.png)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/mpous/helium-data-hotspot)
 
-Follow the instructions, click `Add a Device` and flash an SD card with that balenaOS image
+Follow the instructions, click `Add a Device` and flash a SD card with that balenaOS image
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: https://www.reddit.com/r/grammar/comments/jmrdr/comment/c2ddj9z/?utm_source=share&utm_medium=web2x&context=3
```suggestion
Follow the instructions, click `Add a Device` and flash an SD card with that balenaOS image
```

File: docs/network-iot/hotspots-iot/data-only/dragino-pg1301.mdx [hunk 6]
```diff
@@ -178,7 +178,7 @@ In the below example, you see two values (DCA6328XXX6F and DCA6328XXX70) because
 wired and wireless interface on the Raspberry Pi. It doesn't matter which one you use, as all MAC
 addresses are globally unique.
 
-The goal here is to guarantee a unique gateway ID in the network.
+The goal here is to guarantee a unique gateway ID in the Network.
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/dragino-pg1301.mdx]: we shouldn't capitalize every instance of network

File: docs/network-iot/hotspots-iot/data-only/rak-concentrators.mdx [hunk 7]
```diff
@@ -55,18 +55,18 @@ if you are already familiar with the Raspberry Pi and the RAK concentrator modul
 
 #### Option 2
 
-Follow the Quick Start Guide for your concentrator on the
+Follow the **Quick Start Guide** for your concentrator on the
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/rak-concentrators.mdx]: this doesn't need to be bold

File: docs/network-iot/hotspots-iot/data-only/rak-concentrators.mdx [hunk 8]
```diff
@@ -87,7 +87,7 @@ choose the proper plan according to your country. To finish, press OK.
 </figure>
 <br />
 
-### Verify If The Packet Forwarder Is Running
+### Verify if the Packet Forwarder is running
```
Comment (jthiller) [inline docs/network-iot/hotspots-iot/data-only/rak-concentrators.mdx]: same titlecase rules apply here

**Review by jthiller — CHANGES_REQUESTED** [review]

File: docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx (hunk 3 above)
Comment (Dumpling-girl) [reply docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx]: It was in the original text, but I will do all lower case letters.

**Comment by Dumpling-girl** [comment]
@jthiller I updated the docs after your feedback.

**Review by jthiller — DISMISSED** [review]

**Review by jthiller — APPROVED** [review]

**Pull request merged**